### DISCUSSION TITLE
First simple test on Linux

### DISCUSSION THREAD

**Discussion by nicolasr75** [dfinke/ImportExcel#1318] [Show and tell]
Used this simple test script to prove that I can easily fill my Excel report templates on Linux:

    Import-Module ImportExcel

    $filename = [System.IO.Path]::Combine($PSScriptRoot, "EnergyReport.xlsx")

    $excel = Open-ExcelPackage -Path $filename

    $ws = $excel.Workbook.Worksheets["Sheet 1"]

    $ws.Cells["B1"].Value = "Hello Excel from ImportExcel"

    $excel.Save()

Used PowerShell 7.3.0 on Debian 10.

**Comment by dfinke** [comment]
Good stuff. You can also use `Close-ExcelPackage $excel` which saves the changes, plus it has other parameters that are useful.
